### ISSUE TITLE
Deb Repo for Ubuntu Bionic Users Missing

### ISSUE THREAD

**Issue by cathalgarvey** [RadeonOpenCompute/ROCm_Documentation#21]
Hi folks - I upgraded to Bionic, after checking the ROCm docs first to ensure it was supported on 18.04 ("bionic"). However, after re-enabling my apt-source for ROCm, which now points to "bionic", it errors out saying there's no Release file.

Sure enough, when I open the repository in my browser I see only Xenial.

Should I use Xenial for both distro versions? Or, is the Bionic repository a work-in-progress?

**Comment by jlgreathouse** [comment]
Yes, please use Xenial for both (since the files are the same for 16.04.5 LTS and 18.04.1 LTS).
